src/video/SDL_video.c

```diff
@@ -358,6 +358,20 @@ static void SDL_DestroyWindowTexture(SDL_VideoDevice *unused, SDL_Window *window
     SDL_free(data);
 }
 
+int SDL_SetVSyncWindowTexture(SDL_Window *window, int vsync)
+{
+    SDL_WindowTextureData *data;
+
+    data = SDL_SetWindowData(window, SDL_WINDOWTEXTUREDATA, NULL);
```

Comment:
Did you mean `SDL_GetWindowData(window, SDL_WINDOWTEXTUREDATA)`?

Reply:
yes of course :)
how is this even possibly working!

Reply:
maybe because it returns the last value ...